Fix missing Zipkin error messages

hananiel opened the pull request:
Currently a Zipkin error tag of type string shows up in
Jaeger UI as an error.message tag with the value "False"
instead of the string value. The sanitizer is missing
annotationType on the new tag that holds error.message

Resolves #890 

Signed-off-by: Hananiel Sarella <hananiel@gmail.com>

yurishkuro commented:
@hananiel thanks for the fix! Please amend your commit to make it signed: `git commit -s --amend`, and then run `make fmt` and commit again with `-s` flag.

michael-barker commented:
@yurishkuro This is the fix for the [question I recently asked](https://github.com/jaegertracing/jaeger/pull/327#issuecomment-400152422) on the original pull request.  We'll get the PR fixed tomorrow.

jpkrohling commented:
@hananiel, I changed the title and added a "Resolves" line to your PR description, so that the issue is automatically closed. I hope you don't mind!

hananiel commented:
Thanks Juraci!

On Thu, Jun 28, 2018 at 5:00 AM Juraci Paixão Kröhling <
notifications@github.com> wrote:

> @hananiel <https://github.com/hananiel>, I changed the title and added a
> "Resolves" line to your PR description, so that the issue is automatically
> closed. I hope you don't mind!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jaegertracing/jaeger/pull/891#issuecomment-400964574>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA2Li6nIclJNC37CQTukOFBnYH7CXRjsks5uBJsggaJpZM4U6WVF>
> .
>
